### PR TITLE
Added Indicator Fields

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
related to: https://github.com/demisto/etc/issues/17949

## Description
Added new indicator fields:
- Username
- Account Type
- Blocked
- Display Name
- Groups
- Organizational Unit (OU)
- Email Address
- Detection Engines
- Positive Detections
- Internal
- Malware Family
- CVSS
- CVE Description
- Published
- Modified
- Admin Email
- Admin Name
- Admin Phone
- Admin Country
- Creation Date
- DNS
- Domain Status
- Expiration Date
- Name Servers
- Organization
- Registrant Email
- Registrant Name
- Registrant Phone
- Registrant Country
- Registrar Abuse Email
- Registrar Abuse Phone
- Registrar Name
- Subdomains
- Updated Date
- Associated File Names
- Quarantined
- MD5
- SHA1
- SHA256
- SHA512
- SSDeep
- File Type
- File Extension
- Signature Authentihash
- Signature Copyright
- Signature Description
- Signature File Version
- Signature Internal Name
- Signature Original Name 
- Signed
- Address
- Entry ID
- Tags
- Actor
- Size
- Path
- IP Address
- MAC Address
- DHCP Server
- Operating System
- Operating System Version
- BIOS Version
- Device Model
- Memory
- Processors
- Processor
- ID
- Domain Name
- ASN
- Geo Country
- Geo Location
- Hostname
- Name

### CODE REVIEW

File: IncidentFields/incidentfield-asn.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "type": "shortText",
+  "breachScript": "",
+  "editForm": true,
+  "description": "",
+  "associatedToAll": false,
+  "shouldCommit": false,
+  "shouldPublish": false,
+  "associatedTypes": [
+    "IP",
+    "IP_tmp"
```
Comment (hirshbergshachar) [inline IncidentFields/incidentfield-asn.json]: @DeanArbel what is this type?

Comment (DeanArbel) [reply IncidentFields/incidentfield-asn.json]: that's a placeholder from before export was added, I'll remove it

**Comment by DeanArbel** [comment]
> Overall looks fine. There's duplication for existing indicator fields that I've added (md5,sha1,sha256,ssdeep, associated file names, sha512) Please check those.. Also we need to think if this is the convention we want to indicator fields (`incidentfield-name`) and not (`indicatorfield-name`), need to also check that the later is compatible in the server-side @yuvalbenshalom @anara123 what do you think?

Following the discovery that there is no support for this server side, I believe this can be reviewed on its own. What do you think @hirshbergshachar ?

**Comment by DeanArbel** [comment]
Added @anara123 as a reviewer to approve changed made to `content_creator.py` and to force merge

**Comment by DeanArbel** [comment]
@anara123 tested manually, and as expected the new indicator fields appear in the content.zip (see https://26090-60525392-gh.circle-artifacts.com/0/artifacts/content_new.zip)

**Comment by yardensade** [comment]
This PR broke circle builds since yesterday.
Hence we reverted it, please revisit the changes here carefully.

**Pull request merged**